pkg/osquery/tables/execparsers/apt/parser.go

```diff
@@ -0,0 +1,41 @@
+package apt
+
+import (
+	"bufio"
+	//"fmt"
+	"io"
+	"strings"
+)
+
+func aptParse(reader io.Reader) (any, error) {
+	results := make([]map[string]string, 0)
+
+	scanner := bufio.NewScanner(reader)
+	for scanner.Scan() {
+		line := scanner.Text()
+		pair := strings.Split(line, "/")
```

Comment:
I'll take this. But FYI it might be slightly cleaner using a regex match on the line